### DISCUSSION TITLE
GitHub Linguist Is Ignoring The Directory That I Want To Include

### DISCUSSION THREAD

**Discussion by leimao** [github-linguist/linguist#7166] [Q&A] [answered]
This repo should be a CUDA repo but is reported as a Dockerfile repo: 
https://github.com/leimao/CUTLASS-Examples

I tried playing with the GitHub linguist a little bit locally. It seems that the `examples` directory, which contains all the critical source code, is ignored.

How should we fix the problem? Thank you.

**Comment by lildude** [comment, marked as answer]
Linguist considers the `examples` directory as documentation:

https://github.com/github-linguist/linguist/blob/5fad8d57605a914026a65b0e3ff6815d739944de/lib/linguist/documentation.yml#L17

You can use the [`-linguist-documentation` override](https://github.com/github-linguist/linguist/blob/main/docs/overrides.md#documentation) to override this.
